### DISCUSSION TITLE
Windows 10 build error: "reading zip file"

### DISCUSSION THREAD

**Discussion by BrianHanke** [AcademySoftwareFoundation/OpenRV#192] [Build Issues] [answered]
Making progress on getting the latest Open RV to build, but I've got a new error now. Any suggestions how to fix this? My setup should be pretty standard, had to add the aclocal and nasm fixes to get this far, but other than that it's just a vanilla build. Thanks!

"C:\OpenRV\_build\main_executable.vcxproj" (default target) (1) ->
       "C:\OpenRV\_build\src\bin\apps\rv\rv.vcxproj" (default target) (3) ->
       "C:\OpenRV\_build\executables_with_plugins.vcxproj" (default target) (10) ->
       "C:\OpenRV\_build\src\bin\imgtools\makeMovieIOformats\makeMovieIOformats.vcxproj" (default target) (15) ->
       "C:\OpenRV\_build\installed_packages.vcxproj" (default target) (18) ->
       "C:\OpenRV\_build\src\bin\apps\rvpkg\rvinstall.vcxproj" (default target) (21) ->
       (CustomBuild target) ->
         CUSTOMBUILD : error : reading zip file C:/OpenRV/_build/stage/app/PlugIns/Packages/annotate-1.16.rvpkg [C:\OpenRV\_build\src\bin\apps\rvpkg\rvinstall.vcxproj] C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(241,5): error MSB8066: Custom build for 'C:\OpenRV\_build\CMakeFiles\f434ed987c638a454fba71151c1894c6\rvinstall.rule;C:\OpenRV\_build\CMakeFiles\a3b6ae90bfb4c6c01dcffc11bf00eda8\rvinstall.rule;C:\OpenRV\src\bin\apps\rvpkg\CMakeLists.txt' exited with code -1. [C:\OpenRV\_build\src\bin\apps\rvpkg\rvinstall.vcxproj]

**Comment by rogernelson** [comment]
Sorry, we had a commit that broke the build.  A revert is currently in progress:
https://github.com/AcademySoftwareFoundation/OpenRV/pull/191

**Comment by rogernelson** [comment, marked as answer]
It's reverted now.  If you pull the latest changes, you should be able to build again.  Sorry for the inconvenience.

**Reply by BrianHanke** [reply]
Working great now, thanks!

**Reply by apanner** [reply]
HI .. @BrianHanke .. Could you please guide me build for windows.. I've been trying build keep getting error cmake..
 I ve installed all dependences but still not able to build one..  please list out the steps.  it would be really helpful.. 
 
  cmake -B _build -G "Visual Studio 16 2019" -A x64 -DCMAKE_BUILD_TYPE=Release -DRV_DEPS_WIN_PERL_ROOT=C:\Strawberry\perl\bin -DRV_DEPS_QT5_LOCATION=D:\Qt1\5.15.2\
>
-- Selecting Windows SDK version 10.0.22000.0 to target Windows 10.0.22621.
-- Build type: Release
PROJECT DIR D:/OpenRV
-- Building RV for Microsoft Windows
-- Using build branch: main
-- Using build hash: 3b509ab
-- RV_STAGE_ROOT_DIR: D:/OpenRV/_build/stage/app
-- RV_STAGE_BIN_DIR: D:/OpenRV/_build/stage/app/bin
-- RV_STAGE_LIB_DIR: D:/OpenRV/_build/stage/app/lib
-- RV_STAGE_INCLUDE_DIR: D:/OpenRV/_build/stage/app/include
-- RV_STAGE_SRC_DIR: D:/OpenRV/_build/stage/app/src
-- RV_STAGE_RESOURCES_DIR: D:/OpenRV/_build/stage/app/resources
-- RV_STAGE_PLUGINS_DIR: D:/OpenRV/_build/stage/app/PlugIns
-- RV_STAGE_PLUGINS_CONFIGFILES_DIR: D:/OpenRV/_build/stage/app/PlugIns/ConfigFiles
-- RV_STAGE_PLUGINS_IMAGEFORMATS_DIR: D:/OpenRV/_build/stage/app/PlugIns/ImageFormats
-- RV_STAGE_PLUGINS_MOVIEFORMATS_DIR: D:/OpenRV/_build/stage/app/PlugIns/MovieFormats
-- RV_STAGE_PLUGINS_MU_DIR: D:/OpenRV/_build/stage/app/PlugIns/Mu
-- RV_STAGE_PLUGINS_NODES_DIR: D:/OpenRV/_build/stage/app/PlugIns/Nodes
-- RV_STAGE_PLUGINS_OIIO_DIR: D:/OpenRV/_build/stage/app/PlugIns/OIIO
-- RV_STAGE_PLUGINS_PACKAGES_DIR: D:/OpenRV/_build/stage/app/PlugIns/Packages
-- RV_STAGE_PLUGINS_PROFILES_DIR: D:/OpenRV/_build/stage/app/PlugIns/Profiles
-- RV_STAGE_PLUGINS_PYTHON_DIR: D:/OpenRV/_build/stage/app/PlugIns/Python
-- RV_STAGE_PLUGINS_QT_DIR: D:/OpenRV/_build/stage/app/PlugIns/Qt
-- RV_STAGE_PLUGINS_SUPPORTFILES_DIR: D:/OpenRV/_build/stage/app/PlugIns/SupportFiles
-- Updating submodules
CMake Error at cmake/dependencies/qt5.cmake:50 (FIND_PACKAGE):
  By not providing "FindQt5.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Qt5", but
  CMake did not find one.

  Could not find a package configuration file provided by "Qt5" with any of
  the following names:

    Qt5Config.cmake
    qt5-config.cmake

  Add the installation prefix of "Qt5" to CMAKE_PREFIX_PATH or set "Qt5_DIR"
  to a directory containing one of the above files.  If "Qt5" provides a
  separate development package or SDK, be sure it has been installed.
Call Stack (most recent call first):
  cmake/dependencies/CMakeLists.txt:36 (INCLUDE)


-- Configuring incomplete, errors occurred!

 
Thanks

**Comment by BrianHanke** [comment]
@apanner Hmm, it seems like you don't have Qt installed or you have the wrong Qt path specified.
